packages/nimble-components/src/rich-text-editor/styles.ts

```diff
@@ -1,12 +1,158 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderHoverColor,
+    borderRgbPartialColor,
+    borderWidth,
+    smallDelay,
+    standardPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        block-size: 100%;
+        flex-direction: column;
+        position: relative;
+        --ni-private-hover-indicator-width: calc(${borderWidth} + 1px);
+        ${
+            /**
+             * Min inline size to accommodate all the possible buttons in the toolbar and max inline size is referred from the visual design.
+             */ ''
+        }
+        min-inline-size: 360px;
+        max-inline-size: 720px;
+    }
+
+    .container {
+        display: flex;
+        flex-direction: column;
+        block-size: inherit;
+        position: relative;
+        max-block-size: 100%;
+        border: ${borderWidth} solid rgba(${borderRgbPartialColor}, 0.3);
+    }
+
+    .container::after {
+        display: block;
+        content: ' ';
+        position: absolute;
+        bottom: calc(-1 * ${borderWidth});
+        inline-size: 0px;
+        block-size: 0px;
+        left: 50%;
+        transform: translate(-50%, 50%);
+        border-bottom: ${borderHoverColor}
+            var(--ni-private-hover-indicator-width) solid;
+        transition: width ${smallDelay} ease-in;
+    }
+
+    .container:focus-within {
+        border-bottom-color: ${borderHoverColor};
+    }
+
+    @media (prefers-reduced-motion) {
+        .container::after {
+            transition-duration: 0s;
+        }
+    }
+
+    :host(:hover) .container::after {
+        inline-size: 100%;
+    }
+
+    .editor {
+        border: ${borderWidth} solid transparent;
+        border-radius: 0px;
+        block-size: calc(100% - 44px);
```

Comment:
What is this 44px? Should this be expressed in terms of a private CSS variable you define?

Reply:
Earlier I had the height of the footer section as 44px, so I added the calc to have the remaining height for the editor container. But it should be 40px. As suggested, I created a private variable and used it in both the places.